Client/RecentItemsHelper.swift

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+
+class RecentItemsHelper {
+    
+    /// Filter items that are a older than some number of days.
```

Comment:
```suggestion
    /// Filter `RecentlySavedItem`s that are older than a given number of days.
```

Reply:
I'll fix this in the iPad PR 😬 Should be coming shortly after